content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -135,29 +134,41 @@ For pods, this is any pod that has a deletion timestamp set.
 
 ### Topology information {#topology}
 
-{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
-
-{{< note >}}
-The topology field in EndpointSlices has been deprecated and will be removed in
-a future release. A new `nodeName` field will be used instead of setting
-`kubernetes.io/hostname` in topology. It was determined that other topology
-fields covering zone and region would be better represented as EndpointSlice
-labels that would apply to all endpoints within the EndpointSlice.
-{{< /note >}}
-
 Each endpoint within an EndpointSlice can contain relevant topology information.
 This is used to indicate where an endpoint is, containing information about the
-corresponding Node, zone, and region. When the values are available, the
-control plane sets the following Topology labels for EndpointSlices:
+corresponding Node, zone. When the values are available, the
```

Comment:
nit: Possibly reword?
Each endpoint within an EndpointSlice can contain relevant topology information.
The topology information includes the location of the endpoint and information about the corresponding Node and zone.

Reply:
Thanks for the suggestion!